posts/2022-03-21-synchronous-mutations.md

```diff
@@ -22,21 +19,23 @@ Vendia would return a transaction ID, requiring clients to either poll the ledge
 
 ## Improvement
 
-To address these issues we are introducing new APIs that give you the ability to execute synchronous mutations. Using the new APIs, you will have a choice in whether to make a synchronous or asynchronous mutation. If you make a synchronous mutation, Vendia returns the result once the transaction has been committed and ledgered on the local node. The transaction is guaranteed to be eventually replicated to all the nodes in the Uni.
+To address these issues we are introducing new APIs that give you the ability to execute synchronous mutations. Using the new APIs, you will have a choice in whether to make a synchronous or asynchronous mutation. If you make a synchronous mutation, you have the choice of four different consistency modes. Vendia returns the result to you at different points throughout the consensus phases depending on which of the four different consistency modes you choose. 
 
-## Example
 
-Let’s take a look at some of the changes we made. In the GraphQL Explorer in the [Vendia Share UI](https://share.vendia.net/), you will notice you have access to two different APIs for the same mutation. One that ends in `_async` and one that doesn’t. 
+| Type of Consistency | Dirty Reads | Synchronous E2E Response* | Consistency Guarantees |
+| ----- | ------ | ------ | ------ | 
+|CACHED | Very possible | Very fast | Transactions are written to a local cache on a given Node and can be out of date up to cache timeout. |
+| NODE_COMMITTED | Possible | Fast | Transactions are written to the world state on a given Node but, perhaps, not yet to the global ledger. |
+| NODE_LEDGERED| No | Slow | Transactions are written to the world state on a given Node and the global ledger. |
+| UNI_LEDGERED | No | Slowest | Transactions are written to the world state on all Nodes and the global ledger. |
```

Comment:
Yes - `UNI_LEDGERED` is an option on sync mutations. But thank you for bringing this up because `CACHED` is not value for sync mutations, so I updated this table.